package.json

```diff
@@ -20,7 +24,9 @@
   },
   "scripts": {
     "install": "node selenium/install.js",
-    "test": "./node_modules/.bin/grunt lint; ./node_modules/.bin/grunt test"
+    "test": "./node_modules/.bin/grunt lint; ./node_modules/.bin/grunt test",
+    "lint": "npm run lint:js",
+    "lint:js": "eslint '**/*.{js,jsx}'"
```

Comment:
Might want to double check that this blob works on Windows. I've run into issues with commands before where the quotes used for this mattered to Windows and broke things. 😞